plugins/woocommerce-admin/client/marketplace/components/header-account/header-account.tsx

```diff
@@ -28,7 +28,7 @@ export default function HeaderAccount(): JSX.Element {
 	// component. That component is either an anchor with href if provided or a button that won't accept an href if no href is provided.
 	// Due to early erroring of TypeScript, it only takes the button version into account which doesn't accept href.
 	// eslint-disable-next-line @typescript-eslint/no-explicit-any
-	const accountURL: any = MARKETPLACE_URL + '/my-dashboard/';
+	const accountURL: any = MARKETPLACE_API_HOST + '/my-dashboard/';
```

Comment:
Nitpick: maybe we should call this variable `MARKETPLACE_HOST`. 

`MARKETPLACE_API_HOST + '/my-dashboard/'` looked a little odd to me, considering we're creating a basic webpage URL and not an API endpoint.

Reply:
Good point! Changed in [`6ae83b0` (#39869)](https://github.com/woocommerce/woocommerce/pull/39869/commits/6ae83b08ecb0d4f500996e50136c509ae1cd94f9).